app/javascript/react/screens/App/Plan/PlanActions.js

```diff
@@ -53,6 +53,13 @@ const _getOrchestrationStackActionCreator = (url, playbookScheduleType, task) =>
             type: DOWNLOAD_LOG_COMPLETED,
             payload: task.id
           });
+          dispatch({
+            type: V2V_NOTIFICATION_ADD,
+            message: e.error.message,
+            notificationType: 'error',
+            persistent: true,
+            actionEnabled: false
+          });
           reject(e);
```

Comment:
This will result with duplicate error messages as we started getting error notification for free when we moved away from using `axios` to make requests to the MIQ API.  [(review comment)](https://github.com/ManageIQ/miq_v2v_ui_plugin/pull/442#discussion_r204040854)